app/Console/Commands/User/ChangeEmail.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Console\Commands\User;
+
+use App\User;
+use Illuminate\Console\Command;
+use App\Jobs\UserVerificationCreateTokenAndSendJob;
+
+class ChangeEmail extends Command
+{
+    protected $signature = 'wbs-user:change-email';
+
+    protected $description = 'Change e-mail address for user';
+
+    public function handle(): int
+    {
+        do {
+            $emailOld = $this->ask('Current user address');
+            $user = User::whereEmail($emailOld)->first();
+
+            if ($user) {
+                $this->line("Found a user for '$emailOld'");
+            } else {
+                $this->warn("Did not find a user for '$emailOld'. Please try again.");
+            }
+        } while(!$user);
```

Comment:
Can one Ctrl+C out of such a loop as expected?

Reply:
yes, you can abort it safely anytime before confirming the confirmation prompt